cmake/Install.cmake

```diff
@@ -361,12 +361,14 @@ install(
 set(RULES_XLS Rules22-1-0-to-22-2-0.md)
 install(FILES "${PROJECT_SOURCE_DIR}/release/Bugreprt.txt" DESTINATION "./")
 install(FILES "${PROJECT_SOURCE_DIR}/release/favicon.png" DESTINATION "./")
-install(FILES "${PROJECT_SOURCE_DIR}/release/readme.html" DESTINATION "./")
-install(FILES "${PROJECT_SOURCE_DIR}/release/Deprecation.html" DESTINATION "./")
+configure_file("${PROJECT_SOURCE_DIR}/release/readme.in.html" "${PROJECT_BINARY_DIR}/release/readme.html" @ONLY)
```

Comment:
Just a quick configure on these files using variables already available in the build system at this point.